### DISCUSSION TITLE
create document inside repository

### DISCUSSION THREAD

**Discussion by Asbaharoon** [community/community#29410] [Other Feature Feedback, Questions, & Ideas]
![Github picture](https://user-images.githubusercontent.com/98842320/184531472-cb21fb48-8f69-48a8-8b75-55289c5d3592.jpg)
Dears my friend 

can any one help me how i create doc which i clear it and i Show it inside the red circle ???,

**Comment by Asbaharoon** [comment]
i hope anybody reply to my inquiry

**Comment by thanhngan22** [comment]
you can refer more about it here https://stackoverflow.com/questions/12258399/how-do-i-create-a-folder-in-a-github-repository

**Reply by Asbaharoon** [reply]
thank you very much my friend and the problem was solved with me

**Comment by ritayong99** [comment]
Any idea how we can create a doc inside repository t? i wanna knwo to .
[ShowBox](https://showbox.red/) [Tutuapp](https://tutuapp.win/) [Mobdro](https://mobdro.onl/)

**Reply by Asbaharoon** [reply]
https://stackoverflow.com/questions/12258399/how-do-i-create-a-folder-in-a-github-repository
please open the below link and follow instructions
